vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -34,7 +34,7 @@ vic_cesm_put_data()
 {
     extern all_vars_struct    *all_vars;
     extern atmos_data_struct  *atmos;
-    extern dmy_struct          dmy;
+    extern dmy_struct          dmy_current;
```

Comment:
Whereas the classic and image driver use  a variable `dmy` that is an array of `dmy_struct`s, this change introduces a single `dmy_struct` whose name indicates that this is the date/time of the current timestep and is not an array.